### ISSUE TITLE
how to install investpy's development version?

### ISSUE THREAD

**Issue by alvarobartt** [alvarobartt/investpy#138]
The **installation steps of the investpy's development version are proposed to be included in the README.md** file since a lot of functions are currently under development which include new data. So on, the addition of the development version steps **will ease investpy's development since the users that propose issues can test their functionality once implemented but before it is merged into the master branch and updated at PyPI**.

**Comment by alvarobartt** [comment]
Note that to install development version from Github, the following command is required:

`pip install git+https://github.com/alvarobartt/investpy.git@developer`
